projects/elements/src/lib/lazy-elements/lazy-elements-loader.service.ts

```diff
@@ -2,42 +2,68 @@ import { Injectable } from '@angular/core';
 
 const LOG_PREFIX = '@angular-extensions/elements';
 
+export interface ElementConfig {
+  tag: string;
+  url: string;
+}
+
 @Injectable({
   providedIn: 'root'
 })
 export class LazyElementsLoaderService {
   registry: Map<string, Promise<void>> = new Map<string, Promise<void>>();
+  configs: ElementConfig[] = [];
 
   constructor() {}
 
+  addConfigs(configs: ElementConfig[]) {
+    configs.forEach(config => this.addConfig(config));
+  }
+
+  addConfig(config: ElementConfig) {
+    const existingConfig = this.configs.find(c => c.tag === config.tag);
+    if (existingConfig) {
+      console.warn(
+        `${LOG_PREFIX} - ElementConfig for tag '${config.tag}' was previously added, it cannot be added multiple times`
```

Comment:
This would still needlessly spam people with all the tags any time you add the new `forFeature` ...

Whenever you add new `forFeature`, we inject ` @Optional() @Inject(ELEMENTS_CONFIG) configs = []`

which gives us all the already injected configs, so we will just spam console with warnings...

Would getting the last config in the module be a better aproach? so like `addConfig(configs[configs.lenght - 1])`

Reply:
makes sense! 👍 Done